ci-operator/config/openshift/hive/openshift-hive-master.yaml

```diff
@@ -106,12 +114,31 @@ tests:
       - env: HIVE_IMAGE
         name: hive
       - env: RELEASE_IMAGE
-        name: release:latest
+        name: release:e2e-spoke
       from: src
       resources:
         requests:
           cpu: 100m
     workflow: generic-claim
+- as: e2e-4.9
+  steps:
+    cluster_profile: aws
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:e2e-hub-4.9
+    test:
+    - as: test
+      cli: latest
+      commands: CLOUD=aws make test-e2e
+      dependencies:
+      - env: HIVE_IMAGE
+        name: hive
+      - env: RELEASE_IMAGE
+        name: release:e2e-spoke
+      from: src
```

Comment:
Is this `from` right?

Reply:
This chooses what image runs your tests. So, you clone the source then run `CLOUD=aws make test-e2e` - only you know if that's right

Reply:
Cool, thanks. Yes, I believe that's what we want.